mw-webapp/src/component/table/Table.module.scss

```diff
@@ -16,6 +16,21 @@ $cell-max-width: 600px;
   background: var(--primaryBackgroundColor);
   background-clip: padding-box;
   font-weight: bold;
+
+  // make createdAt fill minimum width
+  &:nth-child(1) {
+    width: 0;
+  }
+
+  // make lastUpdate fill minimum width
+  &:nth-child(2) {
+    width: 0;
+  }
+
+  // make favorites fill minimum width
+  &:nth-child(7) {
```

Comment:
Maybe better to use last-child because in the future we can delte or add columns but favorite column is always at the end

Reply:
used nth-child for consistency (3xnt looks better than first + nth + last)